### ISSUE TITLE
Extindeti paginile personale cu continut

### ISSUE THREAD

**Issue by ciprian-cucu** [ciprian-cucu/pabd#10]
Pentru acest issue, fiecare student va trebui sa faca modificari template-ului (fisierului twig).

Pagina voastra va trebui sa contina minim doua paragafe de text, o lista neordonata si o imagine personala.

De asemenea creati si atasati un fisier css pentru a formata continutul. 
Pagina finala va trebui sa aiba o culoare de fundal, spatiul util (cu continutul) sa nu depaseasca 80% din dimensiunea ecranului si sa fie centrat.
Imaginea trebuie sa fie inclusa in continut in asa fel incat sa perimta text langa ea (fie in dreapta fie in stanga).

Pentru detalii consultati: 
http://symfony.com/doc/master/book/templating.html
http://symfony.com/doc/current/cookbook/assetic/asset_management.html

**Comment by raicabogdan** [comment]
La css e putin ciudat, trebuie facut manual? Altfel nu am gasit cum sa il fac sa citeasca locatia corect din Resourses.

**Comment by ciprian-cucu** [comment]
poti pune fisierul css in folderul web.
in mod normal se lucreaza cu 'bundles' - care grupeaza toate fisierele unui anumit modul.
insa pentru pagini statice, in mod implicit assetic cauta in web care este de fapt folderul 'web-root' din punctul de vedere al accesului la aplicatie.
exista si o posibilitate de a crea symlinks, si atunci putem pune fisierele in alta parte, insa nu cred a e cazul sa ne complicam.

**Comment by raicabogdan** [comment]
Yup, asta am observat dar "pare" neorganizat asa aruncate in web.
